doc/source/getting_started/installing_conda.rst

```diff
@@ -0,0 +1,198 @@
+See the `Managing conda environments`_ documentation for further information.
+
+.. note::
+    If you want to use `Spyder`_ as integrated development environment (IDE, see `Development enviroment`_) on Linux or MacOS, you should also install it in the new conda environment and run it from there.
+    Otherwise, Spyder may not able to use the packages in the conda environment:
+
+    .. code-block:: bash
+
+        $ conda install spyder
+
+    On Windows, you can install Spyder in the root conda environment (run ``deactivate`` to get there), but you need to change its default Python interpreter.
+    To do this, open Spyder and use the navigation bar to open "Tools -> Preferences".
+    Click on "Python interpreter" and change the first setting "Select the Python interpreter for all Spyder consoles" from the default setting to "Use the following Python interpreter:".
+    In the text field, fill in the path to the Python executable in your newly created conda environment.
+    For example, if you installed Miniconda (or Anaconda) in ``C:\Programs\Miniconda3``, then the environment's Python interpreter is ``C:\Programs\Miniconda3\envs\odl-py35\bin\python.exe``.
+    You can use the file system browser (symbol to the right of the text field) to find the interpreter on your system.
+
+
+Installing ODL and its dependencies
+===================================
+Install ODL and all its (minimal) dependencies in a ``conda`` environment of your choice by running
+
+.. code-block:: bash
+
+    $ conda install -c odlgroup odl
+
+.. note::
+    To skip the ``-c odlgroup`` option in the future, you can permanently add the ``odlgroup`` conda channel (see `Managing conda channels`_):
+
+    .. code-block:: bash
+
+        $ conda config --append channels odlgroup
+
+    After that, ``conda install odl`` and ``conda update odl`` work without the ``-c`` option.
+    
+.. _installing_odl_conda_extensions:
+
+Extra dependencies
+------------------
+The following packages are optional and extend the functionality of ODL.
+
+- Image and plot displaying capabilities:
+
+  .. code-block:: bash
+
+    $ conda install matplotlib
+
+- Faster FFT back-end using FFTW (currently not in mainstream conda):
+
+  * Install the `FFTW`_ C library version 3 (all possible precisions).
+    Use your Linux package manager for this task or consult the `Windows <http://fftw.org/install/windows.html>`_ or `MacOS <fftw.org/install/mac.html>`_ instructions, respectively.
+
+  * Run
+
+    .. code-block:: bash
+
+        $ pip install pyfftw
+
+- Wavelet transforms (currently not in mainstream conda):
+
+  .. code-block:: bash
+
+    $ pip install pywavelets
+
+- Simple backend for ray transforms:
+
+  .. code-block:: bash
+
+    $ conda install scikit-image
+
+- Fast `ASTRA`_ ray transform backend:
+
+  .. code-block:: bash
+
+    $ conda install -c astra-toolbox astra-toolbox
```

Comment:
How about we leave this and let the astra people sort this out. If 1.9 is released before a fix, we'll get it done.